content/practices/landmark-regions/landmark-regions-practice.html

```diff
@@ -80,91 +80,91 @@ <h2>HTML Sectioning Elements</h2>
             <tr>
               <td><code>section</code></td>
 
-              <td><code>region</code> when it has an accessible name using <code>aria-labelledby</code> or <code>aria-label</code></td>
+              <td><code>region</code>（<code>aria-labelledby</code>または<code>aria-label</code>を使用してアクセシブルな名前がある場合）</td>
             </tr>
           </tbody>
         </table>
       </section>
 
       <section>
-        <h2>General Principles of Landmark Design</h2>
-        <p>Including <strong>all perceivable content</strong> on a page in one of its landmark regions and giving each landmark region a semantically meaningful role is one of the most effective ways of ensuring assistive technology users will not overlook information that is relevant to their needs.</p>
-        <p><strong>Step 1: Identify the logical structure</strong></p>
+        <h2>ランドマークデザインの一般的な原則</h2>
+        <p>ページ上の<strong>すべての知覚可能なコンテンツ</strong>をランドマーク領域のいずれかに含め、各ランドマーク領域に意味的に意味のある役割を与えることは、支援技術の利用者が自分たちのニーズに関連する情報を見逃さないようにする最も効果的な方法の一つです。</p>
+        <p><strong>ステップ1: 論理構造を特定する</strong></p>
 
         <ul>
-          <li>Break the page into perceivable areas of content which designers typically indicate visually using alignment and spacing.</li>
+          <li>ページを、デザイナーが通常、整列と間隔を使用して視覚的に示す知覚可能なコンテンツの領域に分割します。</li>
 
-          <li>Areas can be further defined into logical sub-areas as needed.</li>
+          <li>領域は必要に応じて論理的なサブエリアにさらに定義することができます。</li>
```

Comment:
領域とサブエリアとするにはすわりがよくないと思いますので。

```suggestion
          <li>必要に応じて、領域は論理的なサブ領域にさらに定義できます。</li>
```